types/codemirror/codemirror-panel.d.ts

```diff
@@ -0,0 +1,51 @@
+// Type definitions for CodeMirror
+// Project: https://github.com/marijnh/CodeMirror
+// Definitions by: Nikolaj Kappler <https://github.com/nkappler>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// See docs https://codemirror.net/doc/manual.html#addon_panel
+
+import * as CodeMirror from "codemirror";
+
+declare module "codemirror" {
+    interface Panel {
+        /**Removes the panel from the editor */
+        clear: () => void;
+        /**Notifies panel that height of DOM node has changed */
+        changed: (height?: number) => void;
```

Comment:
It's easier to read with method-like notation, no? Can we lose those extra arrow characters please?

```typescript
interface Panel {
  clear(): void;
  changed(height?: number): void;
}